### PR TITLE
Bug 1533717 - Add a search keyword "Whatever"

### PR DESCRIPTION
**Pull request by kyoshino**
Allow to do custom searches like 

* `Whatever / changed by / foo@bar.org`
* `Whatever / changed after / -1m`

## Bugzilla link

[Bug 1533717 - Add a search keyword "Whatever"](https://bugzilla.mozilla.org/show_bug.cgi?id=1533717)

### CODE REVIEW

**Comment by kyoshino** [comment]
Hmm, this is hitting the old Firefox frontend issue. Will try to find a workaround...

**Comment by kyoshino** [comment]
The script has to be updated once #1239 is merged.

File: Bugzilla/Field.pm [hunk 1]
```diff
@@ -486,6 +486,9 @@ use constant DEFAULT_FIELDS => (
     type    => FIELD_TYPE_USER,
     buglist => 1,
   },
+
+  # Special field that allows to search everything with Custom Search queries
+  {name => 'whatever', desc => 'Whatever'},
```
[the remark this answers is not in the document]
Comment (dklawren) [reply Bugzilla/Field.pm]: The rest looks good and tested fine.

**Comment by kyoshino** [comment]
Renamed the field, and also removed it from `DEFAULT_FIELDS`. It’s just a virtual field so the definition doesn’t have to be stored in the database.

**Comment by kyoshino** [comment]
Er, no, it still has to be a formal field, otherwise you’ll get an error: “Can’t use anything as a field name.“

**Pull request merged**